README.md

```diff
@@ -387,6 +389,25 @@ defradb start --allowed-origins=http://localhost:3000
 
 The catch-all `*` is also a valid origin. 
 
+## Backing up and restoring
+
+It is currently not possible to do a full backup of DefraDB that includes the history of changes through the Merkle DAG. However, DefraDB currently supports a simple backup of the current data state in JSON format that can be used to seed a database or help with transitioning from one DefraDB version to another.
+
```

Comment:
thought: If the above thought is expanded then we can have the cli be:
```
defradb client backup path/to/backup.json
defradb client restore path/to/backup.json
```

Instead of:
```
defradb client backup export path/to/backup.json
defradb client backup import path/to/backup.json
```

Reply:
I had 
```
defradb client export path/to/backup.json
defradb client import path/to/backup.json
```
before and it was suggested that grouping them under a command / api path would be clearer.

Reply:
Hmm, I do like the one with less words to type. But if there seems to be a consensus reached already then will just follow that (also is a non-blocker thought).